Index: srfirewall-2.0a1+svn20140408/bin/srfirewall
===================================================================
--- srfirewall-2.0a1+svn20140408.orig/bin/srfirewall	2014-04-08 18:55:30.209705751 -0600
+++ srfirewall-2.0a1+svn20140408/bin/srfirewall	2014-04-08 19:22:51.369154162 -0600
@@ -22,7 +22,7 @@
 FW_VERSION="2.0 PreAlpha 1"
 
 # Important directory locations
-FWPREFIX="/usr/local"
+FWPREFIX="/usr"
 FWCONFIGDIR="${FWPREFIX}/etc/srfirewall"
 FWLIBDIR="${FWPREFIX}/lib/srfirewall"
 FWBINDIR="${FWPREFIX}/bin"
